### PR TITLE
Added BTL Token to Defaults

### PR DESCRIPTION
**Pull request by deniskashargin**

### CODE REVIEW

File: tokens/tokens-eth.json [hunk 1]
```diff
@@ -10460,5 +10460,38 @@
...
   "youtube"   : ""
 }
+},
+  {
+"symbol"      : "BTL",
+"address"     : "0x92685E93956537c25Bb75D5d47fca4266dd628B8",
+"decimals"    : "4",
+"name"        : "Bitlle Token",
+"ens_address" : "",
+"website"     : "https://bitlle.com/",
```
Comment (ligi) [inline tokens/tokens-eth.json]: Please remove the last slash

**Comment by deniskashargin** [comment]
@ligi  When does added token is shown in the default tokens list?

**Comment by ligi** [comment]
@deniskashargin depends on the implementation - but after now (this merge)

**Comment by deniskashargin** [comment]
Hi  @ligi ,
Thanks but I still not see our token as default.
Any issues?

**Comment by ligi** [comment]
I see BTL in MEW:
selection_205

**Comment by deniskashargin** [comment]
@ligi, that's great, but that's not ours one) This is BTL - Battle. And we're BTL (Bitlle)

**Comment by ligi** [comment]
Then I do not know - it is merged in this repo - maybe the problem is that the symbol is not unique - better ask @tayvano - not sure how duplicates are handled in MEW - but from this repo it should be clear to go.

**Pull request merged**